packages/create-amplify/src/execute_with_logger.ts

```diff
@@ -0,0 +1,30 @@
+import { execa as _execa } from 'execa';
+import { logger } from './logger.js';
+
+/**
+ * Abstracts the execution of a command and pipes outputs/errors to `logger.debug`
+ */
+export const executeWithLogger = async (
+  execa = _execa,
+  projectRoot: string,
```

Comment:
for the context of this function, this is just `cwd`. It doesn't have to be the project root